pytest_httpx/__init__.py

```diff
@@ -1,3 +1,4 @@
+from collections.abc import Generator
```

Comment:
It should be imported from typing as this is for type hinting:

```suggestion
from typing import Generator
```

I would avoid 2 lines of typing imports them, and straight away add it to the line underneath, where List is also imported from typing

Reply:
Could do, but importing `Generator` from `typing` has been deprecated since 3.9.

Reply:
Indeed, as stated in the documentation](https://docs.python.org/3.9/library/typing.html#typing.Generator). However the current proposal is incompatible with python < 3.9.

Given the current status of python version < 3.9, I think we can safely drop the support for python < 3.9 from now on. So instead of importing from typing, we should indeed keep your proposal.

Reply:
Ill drop the support